pkgs/applications/virtualization/colima/default.nix

```
@@ -1,14 +1,20 @@
 { lib
+, stdenv
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
 , lima
+, lima-bin
 , makeWrapper
 , qemu
 , testers
 , colima
 }:
 
+# use lima-bin on darwin to support native macOS virtualization
+# https://github.com/NixOS/nixpkgs/pull/209171
+let lima-drv = if stdenv.isDarwin then lima-bin else lima;
+in
```

```suggestion
, colima
  # use lima-bin on darwin to support native macOS virtualization
  # https://github.com/NixOS/nixpkgs/pull/209171
, lima-drv ? if stdenv.isDarwin then lima-bin else lima
}:

```

I would like this to be configurable, in my opinion.

---

Great suggestion. Thanks!